ceph/ceph_admin/apply_spec.py

```diff
@@ -0,0 +1,74 @@
+"""
+Represent the ceph orch CLI action 'apply' using spec file.
+
+Module to deploy ceph service(s) using orchestration command
+"ceph orch apply -i spec.yaml"
+
+"""
+import logging
+import tempfile
+from typing import Dict
+
+from .common import config_dict_to_string
+from .helper import PrepareSpec
+from .typing_ import ServiceProtocol
+
+LOG = logging.getLogger()
+
+
+class ApplySpec:
```

Comment:
Can we rename it has `ApplySpecMixin` as it provides an added functionality to `orch` class without an initialization method?

Reply:
As we discussed, apply_spec is part Orch.